core/src/main/kotlin/model/Documentable.kt

```diff
@@ -21,21 +22,57 @@ class Package(
     override val name = dri.packageName.orEmpty()
 }
 
-class Class(
+open class Class(
```

Comment:
`interface ClassLike` instead of opening this one

Reply:
Or abstract class, since `Documentable` is already an abstract class.

Reply:
Done.